templates/extra-project.template.yaml

```diff
@@ -0,0 +1,120 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
This is a subset of the full-stack.template.yaml used to register extra projects in the infrastructure